crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -2002,6 +1995,31 @@ impl IndexerStore for PgIndexerStore {
         Ok(())
     }
 
+    async fn restore_display(&self, bytes: bytes::Bytes) -> Result<(), IndexerError> {
+        let cursor = Cursor::new(bytes);
+        let mut csv_reader = ReaderBuilder::new().has_headers(true).from_reader(cursor);
+        let mut displays = vec![];
+        for result in csv_reader.records() {
+            let record = result.map_err(|e| {
+                IndexerError::GcsError(format!("Failed to read display record: {}", e))
+            })?;
+            let new_display = StoredDisplay {
+                object_type: record[0].to_string(),
+                id: hex::decode(&record[1]).map_err(|e| {
+                    IndexerError::GcsError(format!("Failed to decode display id: {}", e))
+                })?,
+                version: record[2].parse().map_err(|e| {
+                    IndexerError::GcsError(format!("Failed to parse display version: {}", e))
+                })?,
+                bcs: hex::decode(&record[3]).map_err(|e| {
+                    IndexerError::GcsError(format!("Failed to decode display bcs: {}", e))
+                })?,
+            };
+            displays.push(new_display);
+        }
```

Comment:
Can we simplify this using the serde functionality from csv? https://docs.rs/csv/latest/csv/#example-with-serde